commercetools.Sdk/Tests/commercetools.Sdk.Serialization.Tests/AttributeDeserializationTests.cs

```diff
@@ -342,6 +342,24 @@ public void DeserializeDateTimeAsTextAttribute()
             var deserialized = serializerService.Deserialize<ProductVariant>(serialized);
             Assert.IsAssignableFrom<Attribute<string>>(deserialized.Attributes[0]);
         }
+        
+        [Fact]
+        public void DeserializeProductWithSetOfSetAttribute()
+        {
+            var serializerService = this.serializationFixture.SerializerService;
+            var productSerialized = File.ReadAllText("Resources/Attributes/SetOfSetAttribute.json");
+            
+            var product = serializerService.Deserialize<Product>(productSerialized);
```

Comment:
I would reduce the json to one Variant with the set-of-set attribute. Makes it simpler to read the test and fixture data